test/basic.html

```diff
@@ -469,7 +516,9 @@
 
         // Wait one tick for observeNodes.
         Polymer.Base.async(function() {
-          form._form.querySelector('input[type=submit]').click();
+          form.$.content.assignedNodes()
+            .find((n) => n.nodeType === Node.ELEMENT_NODE)
```

Comment:
So i am a bit worried a lot of the code in here (assignedNodes, slotchange, etc) doesn't actually work for a hybrid element, and this may be better off in _your_ version of iron-form. WDYT?